src/YoloDev.Expecto.TestSdk/build/net6.0/YoloDev.Expecto.TestSdk.props

```diff
@@ -1,10 +1,26 @@
 <Project>
+
+  <PropertyGroup>
+    <!-- Makes the new platform/runner opt-in feature -->
+    <EnableExpectoRunner Condition=" '$(EnableExpectoRunner)' == '' ">false</EnableExpectoRunner>
+    <!-- Defines the new platform capability -->
+    <IsTestingPlatformApplication>$(EnableExpectoRunner)</IsTestingPlatformApplication>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!--
+      !!! IMPORTANT !!!
+      DO NOT CHANGE THE GUID, IT'S A WELL KNOWN EXTENSION POINT AND THIS EXTENSION NEEDS TO BE REGISTERED AT THE END
+      WE HAVE CODE INSIDE THE TASK 'TestingPlatformEntryPoint' TO ENSURE THE ORDER OF THE REGISTRATION BASED ON THIS GUID
+    -->
+    <TestingPlatformBuilderHook Include="CFC308E5-E1A5-4B99-8002-17EF4D55D043" Condition=" $(GenerateTestingPlatformEntryPoint) == 'true' " >
+      <DisplayName>Expecto</DisplayName>
+      <TypeFullName>YoloDev.Expecto.TestSdk.TestingPlatformBuilderHook</TypeFullName>
```

Comment:
I need to debug that part, it seems that this is not working correctly for F#.

Reply:
Module members may get a + in their name at some point - maybe load this DLL in FSI and check the generated name?